polaris-for-vscode/LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2017-present Shopify Inc.
```

Comment:
Can we just have on LICENSE in the root of the monorepo? Does this get included in the published files?